src/main/java/com/spotify/docker/client/messages/ContainerConfig.java

```diff
@@ -392,6 +400,9 @@ public static Healthcheck create(
       if (retries != null) {
         builder.retries(retries);
       }
+      if (startPeriod != null) {
```

Comment:
You should not need this null check since you marked it `@Nullable` above. If someone sends in a null value, it's no big deal.